demo-angular/package.json

```diff
@@ -38,7 +38,7 @@
     "nativescript-theme-core": "~1.0.2",
```

Comment:
I think the bundle commands are not needed anymore since you can use `tns bundle`
Also, there are new versions of nativescript-dev-webpack, nativescript-dev-webpack so you can bump it. I haven't check all other nativescript dependencies, but probably it's a good change to update them as well.